.github/workflows/test.yml

```diff
@@ -37,9 +37,18 @@ jobs:
         include:
           - container: ghcr.io/oca/oca-ci/py3.6-odoo14.0:latest
             name: test with Odoo
+            exclude: "base_delivery_carrier_label,delivery_carrier_label_batch,delivery_roulier,delivery_roulier_option,delivery_roulier_laposte_fr,delivery_roulier_chronopost_fr,carrier_account_environment"
           - container: ghcr.io/oca/oca-ci/py3.6-ocb14.0:latest
             name: test with OCB
             makepot: "true"
+            exclude: "base_delivery_carrier_label,delivery_carrier_label_batch,delivery_roulier,delivery_roulier_option,delivery_roulier_laposte_fr,delivery_roulier_chronopost_fr,carrier_account_environment"
```

Comment:
Wait, this is going to slow down builds a lot.
Don't we have any better way to handle this? Normally this happens because modules tend to enable features by default. Can't we don anything to prevent that?

Reply:
Yes but there is No Progress an No Response. See https://github.com/OCA/delivery-carrier/pull/643

Reply:
Could we please merge this for now. I really waiting on this. If there is anyone which refactors the false implemented addon base_delivery_carrier_label, also the test flow can be changed again.

Reply:
ping
Also https://github.com/OCA/delivery-carrier/pull/643 will created separate github flow runs.

Reply:
@mt-software-de  Please have a look to https://github.com/OCA/delivery-carrier/pull/854